migration/20170224-2-copy-collection-configuration-into-database.py

```diff
@@ -131,7 +131,7 @@ def convert_content_server(_db, library):
         name="Open Access Content Server"
     )
     library.collections.append(collection)
-    collection.url = url
+    collection.external_integration.url = url
```

Comment:
This should be setting external account id rather than url - that's what got lost from the other migration.